docs/modules/ROOT/nav.adoc

```diff
@@ -284,6 +284,7 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-2020-5.adoc[Payara Community 5.2020.5 Release Notes]
```

Comment:
```suggestion
```
5.2020.5 release notes are already in the General Info section, they shouldn't be duplicated in the History appendix.

Reply:
I'll merge this PR and fix this afterwards with a direct commit.